src/xERC20/contracts/XERC20Lockbox.sol

```diff
@@ -136,6 +136,7 @@ contract XERC20Lockbox is Initializable, IXERC20Lockbox {
         }
 
         XERC20.mint(_to, _amount);
+        XERC20.increaseCrossChainTransferLimit(_amount);
```

Comment:
I think this is incorrect--we don't increase transfer limit until L2 asset hits L1, so better to handle it manually with relayer on the chain from where the deposit came from

Reply:
lockbox will only be on L1 so this is fine

Reply:
let's add a comment here to document that we don't expect this code to run because lockbox won't be on L2 (only on L1) so this is a proxy for checking that this is only L1

Reply:
TODO: let's add the router for L1 (i.e. deposit ultraETHs in lockbox and then call transferRemote)